src/scripts/Tooltip.jsx

```diff
@@ -336,13 +412,16 @@ export default class JoyrideTooltip extends React.Component {
       return output.tooltipComponent;
     }
 
+    const overlayStyles = {
+      cursor: disableOverlay ? 'default' : 'pointer',
+      height: document.body.clientHeight,
+      pointerEvents: this.state.mouseOverHole ? 'none' : 'auto',
```

Comment:
Are you using `pointer-events` to disable any type of click event? I am curious if it behaves properly in non-chrome browsers because I've been burned by them couple times. [Caniuse](http://caniuse.com/#search=pointer-events) says everyone is good to go so I am not really that worried about it anymore.